i18n/create_msg.js

```diff
@@ -76,7 +103,9 @@ langfiles.forEach(langfile => {
 
     fs.writeSync(message_fd, `\n\nBlockly.CatblocksMsgs.locales["${lang_name}"] = {\n`);
     Object.keys(json_object).forEach(key => {
-      fs.writeSync(message_fd, `  "${key}": "${json_object[key]}",\n`);
+      if (langfile === (DEFAULT_LANGUAGE + ".json") || key === "DROPDOWN_NAME") {
+        fs.writeSync(message_fd, `  "${key}": "${json_object[key]}",\n`);
+      }
     });
```

Comment:
If possible shrink output to 
Blockly.CatblocksMsgs.locales = {
 af: {"DROPDOWN_NAME": "Afrikaans"},
ar: { "DROPDOWN_NAME": "Arabic"}
}

and so on

Reply:
simplified output as you can see in the changes of catblocks_msgs.js